### PR TITLE
update: android-ios integration tutorial

### PR DESCRIPTION
**Pull request by danil-pavlov**

### CODE REVIEW

**Comment by danil-pavlov** [comment]
@koshachy, I fixed the issue with nesting directories for code snippets

File: docs/topics/multiplatform-mobile/multiplatform-mobile-integrate-in-existing-app.md [hunk 1]
```diff
@@ -6,18 +6,25 @@ You'll be able to write code and test it for both Android and iOS only once, in
 This tutorial uses a [sample Android application](https://github.com/Kotlin/kmm-integration-sample) with a single screen for entering a username and password.
 The credentials are validated and saved to an in-memory database.
 
-If you aren't familiar with Kotlin Multiplatform Mobile, you can learn how to [create and configure a cross-platform moblie application from scratch](multiplatform-mobile-create-first-app.md) first.
+If you aren't familiar with Kotlin Multiplatform Mobile, you can learn how to [create and configure a cross-platform mobile application from scratch](multiplatform-mobile-create-first-app.md) first.
```
Comment (koshachy) [inline docs/topics/multiplatform-mobile/multiplatform-mobile-integrate-in-existing-app.md:9]:
```suggestion
If you aren't familiar with Kotlin Multiplatform Mobile, learn how to [create and configure a cross-platform mobile application from scratch](multiplatform-mobile-create-first-app.md) first.
```

File: docs/topics/multiplatform-mobile/multiplatform-mobile-integrate-in-existing-app.md [hunk 2]
```diff
@@ -6,18 +6,25 @@ You'll be able to write code and test it for both Android and iOS only once, in
...
    {type="note"}
 
-2. In Android Studio, create a new project from version control: `https://github.com/Kotlin/kmm-integration-sample`.
+2. In Android Studio, create a new project from version control:
+
+   ```text
+   https://github.com/Kotlin/kmm-integration-sample
+   ```
+
+   The `master` branch contains the project's initial state, a simple Android application. You can find the final state
```
[the remark this answers is not in the document]
Comment (koshachy) [reply docs/topics/multiplatform-mobile/multiplatform-mobile-integrate-in-existing-app.md:26]: Let's reduce all these "You can".
To get the final state, switch to the `final` branch of smth like that.

File: docs/topics/multiplatform-mobile/multiplatform-mobile-integrate-in-existing-app.md [hunk 3]
```diff
@@ -74,7 +81,7 @@ You can learn more about the [project structure](multiplatform-mobile-understand
 
 To use cross-platform code in your Android application, connect the shared module to it, move the business logic code there, and make this code cross-platform.
 
-1. Ensure that `compileSdkVersion` and `minSdkVersion` in `build.gradle.kts` of the `shared` module are the same as those in the `build.gradle` of your Android application in the `app` module.  
+1. Ensure that `compileSdk` and `minSdk` in `build.gradle.kts` of the `shared` module are the same as those in the `build.gradle` of your Android application in the `app` module.
```
Comment (koshachy) [inline docs/topics/multiplatform-mobile/multiplatform-mobile-integrate-in-existing-app.md:84]: There is a problem with staging. 
Just want to be sure, that it's OK with rendering this and the markup is not overloaded.

File: docs/topics/multiplatform-mobile/multiplatform-mobile-integrate-in-existing-app.md [hunk 4]
```diff
@@ -122,25 +138,26 @@ You can now extract the business logic code to the Kotlin Multiplatform shared m
...
     ```
 
     ```kotlin
-    //After
+    // After
     return Result.Error(RuntimeException("Error logging in", e))
     ```
 
-2. For email validation, replace the `Patterns` class from the `android.utils` package with a Kotlin regular expression matching the pattern in the `LoginDataValidator` class:
+2. In the `LoginDataValidator` class, for email validation, replace the `Patterns` class from the `android.utils` package with a Kotlin regular expression matching the pattern:
```
Comment (koshachy) [inline docs/topics/multiplatform-mobile/multiplatform-mobile-integrate-in-existing-app.md:160]: Moving **For email validation** in the middle of the sentence make it hard to read.

File: docs/topics/multiplatform-mobile/multiplatform-mobile-integrate-in-existing-app.md [hunk 5]
```diff
@@ -182,29 +199,31 @@ You can learn more about [connecting to platform-specific APIs](multiplatform-co
...
+1. Create a `Utils.kt` file in the `com.jetbrains.simplelogin.shared` package of the `shared/src/commonMain` directory and provide the `expect` declaration:
 
     ```kotlin
     package com.jetbrains.simplelogin.shared
     
     expect fun randomUUID(): String
     ```
 
-2. Create a `Utils.kt` file in the `shared/src/androidMain` directory and provide the `actual` implementation for `randomUUID()` in Android:
+2. Create a `Utils.kt` file in the `com.jetbrains.simplelogin.shared` package of the `shared/src/androidMain` directory and provide the `actual` implementation for `randomUUID()` in Android:
```
Comment (koshachy) [inline docs/topics/multiplatform-mobile/multiplatform-mobile-integrate-in-existing-app.md:210]: the `Utils.kt`?

File: docs/topics/multiplatform-mobile/multiplatform-mobile-integrate-in-existing-app.md (hunk 3 above)
Comment (danil-pavlov) [reply docs/topics/multiplatform-mobile/multiplatform-mobile-integrate-in-existing-app.md]: I removed one of the code items and rearranged others to unload the markup

**Comment by danil-pavlov** [comment]
@koshachy Please take a look again, I improved the text following your suggestions

**Review by koshachy — APPROVED** [review]

**Pull request merged**